### ISSUE TITLE
 module 'moondream' has no attribute 'vl'

### ISSUE THREAD

**Issue by Donaghys** [m87-labs/moondream#177]
Thank you to the author for their work. I encountered an error when running the code according to the example. What is the error?
this is my code

import moondream as md
from PIL import Image

# Initialize with local model path
model = md.vl(model="./weights/moondream-0_5b-int8.mf")

# Load and encode image
image = Image.open("dog.jpg")
result = model.detect(image, "dog")

**Comment by icsy7867** [comment]
I am also getting this error:
```
Traceback (most recent call last):
  File "/app/python/moondream.py", line 1, in <module>
    import moondream as md
  File "/app/python/moondream.py", line 6, in <module>
    model = md.vl(model="/app/python/moondream-2b-int8.mf")
            ^^^^^
AttributeError: partially initialized module 'moondream' has no attribute 'vl' (most likely due to a circular import)
```

I see you closed the issue.  Did you resolve this?

**Comment by Donaghys** [comment]
> I am also getting this error:
> 
> ```
> Traceback (most recent call last):
>   File "/app/python/moondream.py", line 1, in <module>
>     import moondream as md
>   File "/app/python/moondream.py", line 6, in <module>
>     model = md.vl(model="/app/python/moondream-2b-int8.mf")
>             ^^^^^
> AttributeError: partially initialized module 'moondream' has no attribute 'vl' (most likely due to a circular import)
> ```
> 
> I see you closed the issue. Did you resolve this?
Change to a different working directory, as the current directory contains folders with duplicate names, causing Python to fail in parsing the model.

**Comment by icsy7867** [comment]
Fwiw... The hugging face transformers method works, even when using GPU, but there are a lot of cuda dependencies.  Added about 6gb to my docker container size.

**Comment by pratap73** [comment]
hi @Donaghys , i am getting the same error can you please tell, how you did you resolve this error.?

**Comment by icsy7867** [comment]
> hi [@Donaghys](https://github.com/Donaghys) , i am getting the same error can you please tell, how you did you resolve this error.?

I don't know enough about python, but from @Donaghys reply, I decided to play around around the repo.  From the instructions it sounded like i just needed to:

(Applogies on mobile...)
Do the pip install moondream
And then create their example python file.

But what I did to get this to work...

Clone the repo; git clone https://....
cd into the root of the project from the clone
Navigate to ./moondream/clients/python (from inside the moondream folder)

And I tried from this directory and it worked.

I also ran the documented:
poetry install --extras "cpu"

But I'm not sure if that's needed.   I can test more Tuesday.
